src/components/subscription/UpgradeDonationCTA.tsx

```
@@ -0,0 +1,145 @@
+          >
+            <div className="relative overflow-hidden rounded-lg border border-blue-500/20 bg-gradient-to-r from-black/50 via-slate-900/50 to-black/50 backdrop-blur-xl shadow-lg">
+              {/* Subtle accent gradient */}
+              <div className="absolute top-0 left-0 right-0 h-0.5 bg-gradient-to-r from-blue-500/40 via-purple-500/30 to-blue-500/40" />
+              
+              <div className="relative p-3 sm:p-4">
+                {/* Close button */}
+                <button
+                  onClick={handleDismiss}
+                  className="absolute top-2 right-2 sm:top-3 sm:right-3 w-6 h-6 rounded-full bg-white/5 hover:bg-white/10 border border-white/10 flex items-center justify-center transition-all duration-200 group"
+                  aria-label="Dismiss"
+                >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)